firefox-ios/Client/Utils/OhttpManager.swift

```diff
@@ -0,0 +1,162 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Glean
+
+protocol OhttpManager {
```

Comment:
This whole file can be disregarded as it's just stubbing the dependencies we'll have from Glean and A-S

Reply:
[OhttpManager](https://github.com/mozilla-mobile/firefox-ios/blob/main/MozillaRustComponents/Sources/MozillaRustComponentsWrapper/ASOhttpClient/OhttpManager.swift) from AS is in main now 😄